homeassistant/components/publibike/sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Support for PubliBike Public API for bike sharing in Switzerland."""
+
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import PubliBikeDataUpdateCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the PubliBike sensors."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities([EBikeSensor(coordinator), BikeSensor(coordinator)])
+
+
+class PubliBikeSensor(CoordinatorEntity[PubliBikeDataUpdateCoordinator], SensorEntity):
```

Comment:
HA nowadays supports translation entity names and their attributes. Since 2023.8 also services are translatable.

New integrations are required to add support for [name/attributes translation](https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/). Can you please adopt your code to support it?

Reply:
Hey @edenhaus, I had a look at the documentation you linked but tbh I'm a little bit lost as to what exactly (and where) needs to be updated... Are you maybe aware of a PR which I could look at to see an example? (I looked at a few but couldn't really find anything useful...)

Reply:
First of all, new integration needs to use the [new naming format](https://developers.home-assistant.io/docs/core/entity#has_entity_name-true-mandatory-for-new-integrations), which is also required for translating names.

See https://github.com/home-assistant/core/pull/98224 as hopefully good example :)

Reply:
The new naming format is still missing and this is required for merging